bundles/org.openhab.binding.mail/README.md

```diff
@@ -173,5 +205,22 @@ then
     mailActions.sendMail("mail@example.com", "Test subject", "Test message text")
 end
 ```
+:::
+
+::: tab JavaScript
+
+```javascript
+rule "Send Mail with a 'Reference' header; for threaded view in e-mail client"
+when
+    ...
+then
+    val mailActions = actions.get("mail","mail:smtp:sampleserver")
+    mailActions.addHeader("Reference", "<unique-thread-identifier>")
+    mailActions.sendMail("mail@example.com", "Test subject", "Test message text")
+end
```

Comment:
```suggestion
val mailActions = actions.get("mail","mail:smtp:sampleserver")
mailActions.addHeader("Reference", "<unique-thread-identifier>")
mailActions.sendMail("mail@example.com", "Test subject", "Test message text")
```

Reply:
Hi Jan, it is my first request. So I'm not sure I did the correct way to answer on your change request. Siggi.